app/editor/src/features/content/morning-reports/MorningReports.tsx

```diff
@@ -84,7 +84,7 @@ export const MorningReports: React.FC<IMorningReportsProps> = (props) => {
 
   const handleRowClick = (row: TRow<IContentModel>) => {
     if (row.isSelected)
-      navigate(`/morning/reports/combined/${row.original.id}${window.location.search}`);
+      navigate(`/morning/papers/combined/${row.original.id}${window.location.search}`);
```

Comment:
Renaming as the page is not really a morning report.  It is a curation tool for multiple reports that are related to papers.